app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :first_name %><br />
+    <%= f.text_field :first_name, autofocus: true, autocomplete: "first_name" %>
```

Comment:
This doesn't look like a valid autocomplete field. Valid ones: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

Same for the last_name below also.